### PR TITLE
#7Task.alex.moiseyenko 

### PR DESCRIPTION
**Pull request by alexmoiseyenko**
https://travis-ci.org/github/alexmoiseyenko/js-assignments/builds/714198786

### CODE REVIEW

**Comment by KostyaKov** [comment]
In Travis test there are no functions:
- async should resolve Promises and take values step by step
- async should resolve Promises and take values step by step
- async should handle exception during generator work
Can you add them, please?

**Comment by alexmoiseyenko** [comment]
https://travis-ci.org/github/alexmoiseyenko/js-assignments/builds/722866402

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -33,7 +33,18 @@
...
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+    let count = 99;
+    while (count > 1) {
+        yield `${count} bottles of beer on the wall, ${count--} bottles of beer.`;
+        if (count !== 1) {
+            yield `Take one down and pass it around, ${count} bottles of beer on the wall.`;
+        }
+    }
+        yield `Take one down and pass it around, ${count} bottle of beer on the wall.`;
```
Comment (KostyaKov) [inline task/07-yield-tasks.js:43]: it is necessary to observe indents for lines

**Review by KostyaKov — COMMENTED** [review]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**